pom.xml

```diff
@@ -135,6 +153,29 @@
     </profiles>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.felix</groupId>
+                    <artifactId>maven-bundle-plugin</artifactId>
+                    <version>${maven-bundle-plugin.version}</version>
+                    <extensions>true</extensions>
+                    <configuration>
+                        <obrRepository>NONE</obrRepository>
+                        <manifestLocation>${project.build.directory}/META-INF</manifestLocation>
```

Comment:
No reason, I just copied a pattern I have been using for a while but its perfectly fine to leave at the default. I will modify this.